libc/src/__support/FPUtil/ManipulationFunctions.h

```diff
@@ -19,6 +20,7 @@
 #include "src/__support/macros/attributes.h"
 #include "src/__support/macros/optimization.h" // LIBC_UNLIKELY
 
+#include <fenv.h>
```

Comment:
You should include `FEnvImpl.h` instead.  It is where `raise_except_if_required` is defined.